pkg/restclient/volume.go

```diff
@@ -171,3 +171,10 @@ func (c *Client) EditVolume(volname string, req api.VolEditReq) (api.VolumeEditR
 	err := c.post(url, req, http.StatusOK, &resp)
 	return resp, err
 }
+
+// VolumeReset resets volume options
```

Comment:
Suggestion to make this slightly better:

`VolumeReset resets volume options to their default values`